test/integration/policy_report_metric_test.go

```diff
@@ -227,30 +320,34 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test policyreport_info metric", Or
 		By("Checking the policy metric displays nothing")
 		output, err := common.OcHub("set", "env", "-n", ocmNS, insightsClientDeployment, "--list")
 		Expect(err).To(BeNil())
-		fmt.Println("INSIGHTS CLIENT ENV VARIABLES:")
-		fmt.Println(output)
+		log.Info(fmt.Sprintf("INSIGHTS CLIENT ENV VARIABLES:%s", output))
 
 		policyLabel := `policy="` + userNamespace + "." + noncompliantPolicyNameReport + `"`
 		Eventually(func() interface{} {
-			resp, _, err := common.GetWithToken(insightsMetricsURL, strings.TrimSpace(insightsToken))
+			resp, _, err := common.GetWithToken(
+				insightsMetricsURL,
+				strings.TrimSpace(insightsToken),
+			)
 			if err != nil {
 				fmt.Println("ERROR GETTING METRIC:")
 				fmt.Println(err)
+
 				return err
 			}
 			fmt.Println("metric response received:")
 			fmt.Println(resp)
+
 			return resp
 		}, 10*time.Minute, 1).ShouldNot(common.MatchMetricValue(insightsMetricName, policyLabel, "1"))
 	})
 	AfterAll(func() {
 		// unset poll interval
 		_, err := common.OcHub("set", "env", "-n", ocmNS, insightsClientDeployment, "POLL_INTERVAL-")
 		Expect(err).To(BeNil())
-		common.OcHub("delete", "-f", compliantPolicyYamlReport, "-n", userNamespace)
-		common.OcHub("delete", "route", "-n", ocmNS, "-l", insightsMetricsSelector)
-		common.OcHub("delete", "clusterrolebinding", roleBindingName)
-		common.OcHub("delete", "serviceaccount", saName, "-n", userNamespace)
-		common.OcHub("delete", "namespace", "policy-metric-test-compliant")
+		_, _ = common.OcHub("delete", "-f", compliantPolicyYamlReport, "-n", userNamespace)
```

Comment:
These errors should be checked after the `--ignore-not-found` flag is added to the command.